tests/modules/programs/gnome-shell/gnome-shell.nix

```diff
@@ -0,0 +1,94 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  dummy-gnome-shell-extensions = pkgs.runCommand "dummy-package" { } ''
+    mkdir -p $out/share/gnome-shell/extensions/dummy-package
+    touch $out/share/gnome-shell/extensions/dummy-package/test
+  '';
+
+  test-extension = pkgs.runCommand "test-extension" { } ''
+    mkdir -p $out/share/gnome-shell/extensions/test-extension
+    touch $out/share/gnome-shell/extensions/test-extension/test
+  '';
+
+  test-extension-uuid = pkgs.runCommand "test-extension-uuid" {
+    passthru.extensionUuid = "test-extension-uuid";
+  } ''
+    mkdir -p $out/share/gnome-shell/extensions/test-extension-uuid
+    touch $out/share/gnome-shell/extensions/test-extension-uuid/test
+  '';
+
+  test-theme = pkgs.runCommand "test-theme" { } ''
+    mkdir -p $out/share/themes/Test/gnome-shell
+    touch $out/share/themes/Test/gnome-shell/test
+  '';
+
+  expectedEnabledExtensions = [
+    "user-theme@gnome-shell-extensions.gcampax.github.com"
+    "test-extension"
+    "test-extension-uuid"
+  ];
+
+  actualEnabledExtensions = catAttrs "value"
+    config.dconf.settings."org/gnome/shell".enabled-extensions.value;
+
+in {
+  nixpkgs.overlays = [
+    (self: super: {
+      gnome3 = super.gnome3.overrideScope' (gself: gsuper: {
+        gnome-shell-extensions = dummy-gnome-shell-extensions;
+      });
```

Comment:
Is `pkgs.gnome3` an alias to `pkgs.gnome`?
If so, could you use the latter.

Reply:
Indeed, that seems to be the case:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix#L469

Will update.